### PR TITLE
Temporarily disable Node-style logging for service workers

### PR DESCRIPTION
**Pull request by mrbbot**
Hey! 👋 https://github.com/cloudflare/workerd/pull/1294 broke `console.log()`ing in local development for service workers. It looks like this fix for this might be slightly more involved than initially expected. While we work on a permanent solution, this PR reverts the change if service workers are used. In Wrangler, we can re-enable inspector style logging if we detect the bundle is a service worker.

Ref: https://github.com/cloudflare/workerd/issues/1362

### CODE REVIEW

**Review by dom96 — APPROVED** [review]

**Comment by penalosa** [comment]
@mrbbot is there an associated Wrangler PR?

**Comment by abdolrhman** [comment]
i am waiting for this to get merged, i am having a lot of problems with this 
image

**Comment by pencilcheck** [comment]
Cool, once it is merged then we just need to update wrangler I suppose? Thanks

**Pull request merged**
